ios/engine/KMEI/KeymanEngineDemo/MainViewController.swift

```diff
@@ -213,59 +213,30 @@ class MainViewController: UIViewController, UIAlertViewDelegate, TextViewDelegat
 
   // MARK: - Responding to Keyman notifications
   @objc func keyboardDownloadStarted(_ notification: Notification) {
-    let kbInfo = notification.userInfo?[Key.keyboardInfo] as? [AnyHashable: Any]
-    let kbID = kbInfo?[Key.keyboardId] as? String
-    let langID = kbInfo?[Key.languageId] as? String
-    let kbName = kbInfo?[Key.keyboardName] as? String
-    let langName = kbInfo?[Key.languageName] as? String
-    if kbID != nil && langID != nil && kbName != nil && langName != nil {
-      showActivityIndicator()
-    }
+    showActivityIndicator()
```

Comment:
No condition check at all?  Not even pulling the keyboard info from the notification object?

Reply:
I'm not sure why that condition is there in the first place or is necessary. In any case now, the notification should always contain at least one keyboard info.